regression-test/data/nereids_function_p0/fn_test_ip_normal.csv

```diff
@@ -0,0 +1,72 @@
+29;0.0.0.1;2001:0db8:0:0:0:0:0:1
```

Comment:
What's the difference between fn_test_ip_normal.csv and fn_test_ip.csv?

Reply:
normal ip has ordered ips , but fn_test_ip just has 100 ips without ordered